lib/components/user/places/place-editor.tsx

```diff
@@ -135,14 +137,39 @@ class PlaceEditor extends Component<Props> {
     const nameExample = intl.formatMessage({
       id: 'components.PlaceEditor.nameExample'
     })
+    const placeCharacterCount = place.name?.length || 0
+    const characterRemaining = PLACE_NAME_MAX_LENGTH - placeCharacterCount
+    const charactersOverLimit = 0 - characterRemaining
 
     return (
       <div>
         {!isFixed && (
           <>
             <StyledFormGroup validationState={errorStates.name}>
-              <ControlLabel htmlFor="name">
-                <FormattedMessage id="components.PlaceEditor.namePrompt" />
+              <ControlLabel
+                htmlFor="name"
+                style={{
+                  display: 'flex',
+                  flexDirection: 'row',
+                  justifyContent: 'space-between'
+                }}
+              >
+                <>
+                  <FormattedMessage id="components.PlaceEditor.namePrompt" />
+                </>
+                <div style={{ fontWeight: 300 }}>
```

Comment:
div isn't allowed in label I don't think. I think maybe the actual label should just be prompt and the remaining character should be outside the label

Reply:
Whoops, perhaps my local commit didn't get pushed? This should be fixed in 4b8fd0247a0843f3ec30fe4c0a50869a5d3bcc6d